src/boot/efi/meson.build

```diff
@@ -171,6 +171,7 @@ endif
 efi_config_h = configure_file(
         output : 'efi_config.h',
         configuration : efi_conf)
+efi_config_h_dir = meson.current_build_dir()
```

Comment:
I think it is better to move the test at src/boot/efi.

Reply:
All tests for src/fundamental are in src/test, and also src/boot/efi is for efi code. So I think this place is better.

Reply:
Or, src/boot? Then, conditions for adding the test may become simpler. Currently, it is necessary to check the build supports EFI. Otherwise the above CI failure is triggered.